packages/http-caching-proxy/src/__tests__/integration/http-caching-proxy.integration.ts

```diff
@@ -216,6 +230,31 @@ describe('HttpCachingProxy', () => {
     await proxy.stop();
   }
 
+  function getProxyConfig(): AxiosProxyConfig {
```

Comment:
Have you considered calling `getProxyConfig` from inside `makeRequest`, so that `makeRequest` can keep accepting `proxy` configuration as a string URL? That way there is less things to change in our tests.

In longer term, we can contribute an enhancement to Axios to let is support `proxy` specified as a string.